canary/webrtc-c/CMakeLists.txt

```diff
@@ -42,9 +50,13 @@ add_definitions(-DCMAKE_DETECTED_CACERT_PATH)
 include_directories(${cloudwatch_SOURCE_DIR}/aws-cpp-sdk-core/include)
 include_directories(${cloudwatch_SOURCE_DIR}/aws-cpp-sdk-monitoring/include)
 include_directories(${cloudwatch_SOURCE_DIR}/aws-cpp-sdk-logs/include)
+
 include_directories(${webrtc_SOURCE_DIR}/src/include)
 include_directories(${webrtc_SOURCE_DIR}/open-source/include)
 link_directories(${webrtc_SOURCE_DIR}/open-source/lib)
+
+link_directories(${CMAKE_CURRENT_SOURCE_DIR}/build/_deps/lib)
```

Comment:
It looks like this path is `OPEN_SRC_INSTALL_PREFIX` as defined in LN15 above. Let's use that variable directly so we don't have to change it in two places in case we decided to change the install path.

Reply:
Sounds good, testing build now to ensure it still works 👍